app/editor/src/features/content/tool-bar/sections/filter/utils/checkLatestShowOnlyEntry.ts

```diff
@@ -0,0 +1,51 @@
+import { IContentListFilter } from 'features/content/list-view/interfaces';
+import { ContentTypeName } from 'hooks';
+import { IOptionItem } from 'tno-core';
+
+/**
+ * Function that checks the latest entry in the array of selected options and sets the filter accordingly
+ * @param newValues the current array of selected options
+ * @param onChange the function to perform on change of the filter
+ * @param filter the filter to be updated
+ */
+export const checkLatestShowOnlyEntry = (
+  newValues: IOptionItem[],
+  onChange: (filter: IContentListFilter) => void,
+  filter: IContentListFilter,
+) => {
+  if (newValues[newValues.length - 1].value === ContentTypeName.PrintContent) {
```

Comment:
Use your filter enum instead of mixing all things things.  And use a switch statement.